tsconfig.base.json

```diff
@@ -24,6 +24,7 @@
     "strictNullChecks": true,
     "strictPropertyInitialization": true,
     "target": "ESNext",
-    "useUnknownInCatchVariables": true
+    "useUnknownInCatchVariables": true,
+    "removeComments": true
```

Comment:
does this keep license?

Reply:
Nope. It removes all. 
Was looking at size diff which is around 30Kb unminified ESM compared to the unminified CJS